addons/udes_delivery_control/models/picking_extra_fields.py

```diff
@@ -0,0 +1,146 @@
+from odoo import api, fields, models, _
+from ..misc import date_diff, validate_dates
+
+
+class StockPickingDeliveryControl(models.Model):
+    """
+        Model to hold all extra fields of stock.picking object
+    """
+
+    _name = "udes_picking_extras"
+    _description = "UDES Picking Extra Fields"
+
+    picking_id = fields.Many2one(
+        "stock.picking", required=True, index=True, copy=False, ondelete="cascade"
+    )
+
+    # Unloading Fields
+    u_user_id = fields.Many2one("res.users", string="User", help="Picking completed by user")
```

Comment:
Do we not already keep track of that?

Reply:
We do but not on picking level. They are are on line level

Reply:
We are keeping track on line level but not on picking level